platform/machine/packet/console.go

```diff
@@ -16,20 +16,44 @@ package packet
 
 import (
 	"bytes"
+	"fmt"
 	"os"
 
 	"golang.org/x/crypto/ssh"
 )
 
 type console struct {
-	pc   *cluster
-	f    *os.File
-	buf  bytes.Buffer
-	done chan interface{}
+	pc      *cluster
+	f       *os.File
+	buf     bytes.Buffer
+	done    chan interface{}
+	clients map[string]*ssh.Client
```

Comment:
Why the map? - There is only one SSH client per serial console, this is done in `startConsole`.

Reply:
right - for one moment I thought the console was created _before_ the cluster - but it's actually created for each machine :)